.github/ISSUE_TEMPLATE/--weekly-check-in.md

```diff
@@ -40,6 +40,10 @@ Thanks so much `xyz` <!-- Replace `xyz` with the contributors who worked on the
 ## Next Check-in
 If you would like to open the next check-in..Leave a comment below. We are happy to help if its your first time
 
+### Check-in Ideas
+We are collecting a list of check-in ideas here[https://github.com/publiclab/plots2/issues/9480]. If you are looking for a next check-in idea you can browse here to.
```

Comment:
Well done @shefali12-ab!

But this is the full text you're expected to add:

`We are collecting a list of check-in ideas [here](https://github.com/publiclab/plots2/issues/9480). If you have any ideas please leave a comment there or if you are looking for next check-in ideas you could also browse there too.`

And not:

`We are collecting a list of check-in ideas here[https://github.com/publiclab/plots2/issues/9480]. If you are looking for a next check-in idea you can browse here to.`

Looks like you missed a couple of words. Just copy and paste the full text, commit your changes, and push to this same branch again.

Reply:
Hope this is correct? @cesswairimu

Reply:
Hey @PeculiarE Thank you. I have modified the text.

Reply:
Nice. Well done 😄 👍 .